src/txmempool.h

```diff
@@ -672,10 +672,10 @@ class CTxMemPool
 class CCoinsViewMemPool : public CCoinsViewBacked
 {
 protected:
-    CTxMemPool &mempool;
+    const CTxMemPool &mempool;
 
 public:
-    CCoinsViewMemPool(CCoinsView *baseIn, CTxMemPool &mempoolIn);
+    CCoinsViewMemPool(CCoinsView *baseIn, const CTxMemPool &mempoolIn);
```

Comment:
Thanks for the heads-up, I've updated the patch now.  This is not explicitly mentioned in the doc, but the example code block uses the `uint256& hash` style.